src/components/PersonPage/PersonContainer.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { action } from "@storybook/addon-actions";
+import { text, array, number, date } from "@storybook/addon-knobs";
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+
+import PersonContainer from "./PersonContainer";
+
+<Meta title="Library/Containers/Person" component={PersonContainer} />
+
+# PersonContainer
+
+### An Existing Person
+
+<Props of={PersonContainer} />
+
+<Story name="An Existing Person">
+  <PersonContainer
+    testPersonId="283ffb6c5a48"
+  />
+</Story>
+
+
+### A PersonContainer Error
+
+<Props of={PersonContainer} />
+
+<Story name="A Non-Existing Person">
+  <PersonContainer
+    testPersonId="bad-id"
+  />
+</Story>
```

Comment:
Do we know how storybook reacts to a database request?

Reply:
Any way to mock this behavior?

Reply:
Or... I guess once we have the integration repo up-and-running again we can just point these requests to the integration database.

Reply:
Storybook works with the DB request. It pulls and prints just fine. 

I do want to make mocks though, for the Services. I don't know how to do it cleanly though - something like I did seems very clumsy. I am open to suggestions.

Reply:
The "easy way" to do it is just have another parameter at the end with `mockData` that is an object itself. Idk how to do mocks otherwise...

Reply:
for storybook anyway

Reply:
Maybe @hawkticehurst knows?

Reply:
I have done the "easy" way before - the Model Service just checks for mockData and does an early return with whatever object passed to it.  But that doesn't cleanly mesh with the Storybook.  

Perhaps the following?
Containers have optional props for the things that they fetch - e.g. a Person? for a PersonContainer. That is their mock object for Storybook purposes.  And then Services test in a different way.  IIRC it is possible to have an entire mock Firestore instance that spins up locally, that would probably be ideal. Then we don't have to worry about our testing flow avoiding the actual nuts and bolts of the function.

Reply:
Yea I guess using the firestore mock instance is a better solution.. How easy is it I guess?

Reply:
But yes. for now, I say that the mock data can live on the service level.

Reply:
Agree, mock firestore instance would be ideal. But your solution is OK for now.

Reply:
I'm looking into [a potential solution](https://www.npmjs.com/package/mock-firebase-ts) for a mock firebase.  Perhaps we can set it up to run as part of storybook's run script, and have NetworkService swap to using the mock on some environmental variable?